src/main/java/step3/domain/Customer.java

```diff
@@ -0,0 +1,64 @@
+package step3.domain;
+
+import step3.util.Rank;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Customer {
+    private static final LottoMachine lottoMachine = new LottoMachine();
+    private static int resultPrize = 0;
+
+    private HashMap<Rank, Integer> resultRankMap = new HashMap<>();
```

Comment:
```suggestion
    private Map<Rank, Integer> resultRankMap = new HashMap<>();
```

이펙티브 자바 `아이템 64. 객체는 인터페이스를 사용해 참조하라`를 참고해주세요~
다른 부분도(ex. ArrayList -> List)도 같이 확인해주세요 :)

Reply:
https://github.com/next-step/java-lotto/pull/1938/files#r689026235 해당 코멘트의 내용은 확인하셨는데, 해당 코드(HashMap -> Map)은 놓치셨네요 ㅎㅎ